.travis.yml

```diff
@@ -17,12 +17,10 @@ jdk:
   - oraclejdk8
 
 install:
-  - mvn clean install -DskipTests -Dassembly.skipAssembly=true > /dev/null
+  - bash .travis_install.sh
 
 script:
-  - cd $PINOT_MODULE
-  - cp ../.travis_cmd.sh .
-  - bash .travis_cmd.sh
+  - bash .travis_test.sh
 
 after_success:
   - bash <(curl -s https://codecov.io/bash)
```

Comment:
I don't think we need to go overboard for a complex solution right now. But
it will be good to find out how other teams like Helix do this code
coverage on Travis. if they have built some standard practices we can
borrow them.

Thanks
Ravi

On Thu, Jun 2, 2016 at 2:54 PM, Jackie-Jiang <notifications@github.com>
wrote:

> In .travis.yml
> <https://github.com/linkedin/pinot/pull/140#discussion_r65625758>:
>
> >
> >  script:
> > -  - cd $PINOT_MODULE
> > -  - cp ../.travis_cmd.sh .
> > -  - bash .travis_cmd.sh
> > +  - bash .travis_test.sh
> >
> >  after_success:
> >    - bash <(curl -s https://codecov.io/bash)
>
> @jfim <https://github.com/jfim> Sure we can do that, but that will add
> some operational complexity. My point is, for an open source project
> running on a sandbox, nothing could be done by malicious script. If
> malicious script can get any benefits, then that could happen on any box,
> just checkout our code and run any script you want right?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/linkedin/pinot/pull/140/files/0b806a490eba7a6250c08dab0ce795029c2f111e#r65625758>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ADSJ7uUfyECTvfh91z8-8qVUeKiSv6eOks5qH1EqgaJpZM4IsJ6Y>
> .
>